src/main/java/seedu/task/model/task/Duration.java

```diff
@@ -7,7 +7,7 @@
  * Represents a Task's deadline in the task book.
  * Guarantees: immutable; is valid as declared in {@link #isValidDeadline(String)}
  */
-public class Deadline {
+public class Duration {
```

Comment:
I think you refactored and the stuff here changed too? Might wanna take a look at this.